tests/onmount.test.tsx

```diff
@@ -144,7 +144,7 @@ it('mount/unmount test', async () => {
   expect(onUnMountFn).toBeCalledTimes(1)
 })
 
-it('one derived atom, one onMount for the derived one, and one for the regular atom + onUnMount', async () => {
+it('derive chain', async () => {
```

Comment:
oh, we can leave the descriptive version. my list in the comment is just dumb.

Reply:
Aha, Ok